pangeo/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: pangeo
-version: AUTO_POPULATED_BY_CHARTPRESS_AND_TRAVIS
+version: 0.0.1-set.by.chartpress
```

Comment:
This is the default value reset to if using `chartpress --reset`, which is also a valid SemVer2 version to avoid some issues with Helm 3